policyengine_us/variables/gov/states/sc/tax/income/subtractions/retirement_deduction/sc_retirement_deduction_survivors.py

```diff
@@ -0,0 +1,43 @@
+from policyengine_us.model_api import *
+
+
+class sc_retirement_deduction_survivors(Variable):
+    value_type = float
+    entity = Person
+    label = "South Carolina retirement deduction for survivors"
+    defined_for = StateCode.SC
+    unit = USD
+    reference = (
+        "https://www.scstatehouse.gov/code/t12c006.php",  # SECTION 12-6-1170(A)(1)
+        "https://dor.sc.gov/forms-site/Forms/IITPacket_2021.pdf#page=17",
+    )
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.sc.tax.income.subtractions.retirement_deduction
+        p_cap = p.max_amount
+        age = person("age", period)
+        # line 1
+        max_deduction_allowed = where(
+            age >= p.age_threshold,
+            p_cap.older,
+            p_cap.younger,
+        )
+        # line 2
+        military_retirement_pay_survivors = person(
+            "sc_military_deduction_survivors", period
+        )
+        # line 3
+        retirement_deduction_available = max_(
+            max_deduction_allowed - military_retirement_pay_survivors, 0
+        )
+        # line 4
+        retirement_income_survivors = person("pension_survivors", period)
```

Comment:
```suggestion
        retirement_income_survivors = add(tax_unit, period, ["pension_survivors"])
```

Reply:
Because there could be multiple survivors, and each of the survivors is calculated separately, we kept our old code and add tax_unit.sum at the last step.

Reply:
makes sense